src/components/author.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+const Author = styled.a`
```

Comment:
Might want to rename this to avoid conflict in naming the default export for this file.